packages/eos-components/src/utils.js

```diff
@@ -94,6 +94,12 @@ export function getTopicCardSubtitle(node) {
   }
 
   const count = node.children_count || leaves.length;
+
+  // Don't show anything if the number is 0
+  if (count === 0) {
+    return '';
+  }
```

Comment:
I would prefer if you comment the "N resources" line directly from the Vue template, since this will always be the case in master. And ideally add a FIXME comment explaining why it's commented.

Reply:
I've updated the PR to hide the subtitle, just with a comment and the information there.